website/registries/utils.py

```diff
@@ -4,13 +4,20 @@
     'registered_report': 'Registered Report Protocol Preregistration',
 }
 
+REG_CAMPAIGNS_VERSION = {
+    'prereg_challenge': 2,
+    'prereg': 3,
+    'registered_report': 4,
+}
```

Comment:
hm, so does this need to be updated each time we add a new version? or could `get_campaign_schema` just get the latest version?

Reply:
Changed to using `filter(...).order_by('-schema_version').first()`.